device/iot-device-client/src/main/java/com/microsoft/azure/sdk/iot/device/Message.java

```diff
@@ -48,6 +48,7 @@
     /**
      * Used by receiver to Abandon, Reject or Complete the message
      */
+    @SuppressWarnings("FieldCanBeLocal")
     private String lockToken;
```

Comment:
This is used for building a message.

Reply:
q - shouldn't lock token be populated by service, why do we set it when creating a new message?

Reply:
also, I don't see a getter for lockToken, even if we do set it, how do we send it?

Reply:
If the field is set, and never accessed, and isn't exposed to the user, then I'd just delete this field

Reply:
I am unsure about this. It's used in initialize() method, eventually as part of creating a new message.
`inProgressMessages.put(sendResult.getDeliveryTag(), new Message());`

Do you think it's safe to delete? @timtay-microsoft , @abhipsaMisra